src/README.md

```diff
@@ -18,18 +18,20 @@ As a Platform as a Service, or PaaS, Platform.sh automatically manages everythin
 
 Platform.sh covers not only all of your hosting needs but also most of your DevOps needs. It is a simple, single tool that covers the application life-cycle from development to production and scaling.
 
-You only need to write your code, including a few YAML files that specify your desired infrastructure, commit it to Git, and push.  You don't need to setup anything manually. The web server is already setup and configured, as are any database, search engine or cache that you specify.
+You only need to write your code, including a few YAML files that specify your desired infrastructure, commit it to Git, and push.  You don't need to setup anything manually. The web server is already setup and configured, as is any database, search engine or cache that you specify.
 
-Every branch you push is a fully running environment &mdash; complete with your application code, a copy of your database, a copy of your search index, a copy of your user files, everything &mdash; and its automatically generated URL can be sent to stakeholders or to automated CI systems.  It really is "what would my site look like if I merged this to production?"  Every time.
+Every branch you push is a full copy of a production environment&mdash;complete with your application code, a copy of your database, a copy of your search index, a copy of your user files, everything&mdash;and its automatically generated URL can be sent to stakeholders or to automated CI systems.  It really is "what would my site look like if I merged this to production?"  Every time.
```

Comment:
AFAIK mdash should be surrounded by spaces, no?

Also, the text change here alters the meaning.  A branch becomes a copy of production only once, when created.  It can then diverse dramatically.  The key point is that it is a fully self-contained site, not an alternate view on production data.  I think this text change is incorrect.

Reply:
Em-dashes should never be surrounded by spaces! People doing it like — this or like -- this is a weird style quirk.

How about "a fully independent environment" instead? "Fully running" comes off as extremely vague to me

Reply:
I have never heard a specific call on mdashes like that.  Citation?

"Fully independent environment" works for me.

Reply:
It's a thing in some newspaper style guides to surround an em-dash with a space, but [APA style](http://blog.apastyle.org/apastyle/2010/09/computer-editing-tip-em-dashes.html), [MLA style](http://www.mlanet.org/page/mla-style-manual-punctuation), and [Chicago style](https://apvschicago.com/2011/05/em-dashes-and-ellipses-closed-or-spaced.html) all say no spaces.

I typically default to APA style since I've written literally hundreds of pages of APA-formatted papers, and it has heavy overlap with MLA in terms of stylistic choices.